spec/support/unit/helpers/rails_versions.rb

```diff
@@ -38,5 +38,9 @@ def rails_lt_5?
     def rails_5_x?
       rails_version =~ '~> 5.0'
     end
+
+    def rails_gte_5_2?
```

Comment:
Instead of using a plain version conditional, I've started adding methods to [ActiveRecordVersions](https://github.com/thoughtbot/shoulda-matchers/blob/c47e6db7391963d27cb9b45c37d9aa94e53f0590/spec/support/unit/helpers/active_record_versions.rb) that describe the behavior the stuff inside the conditional is for. What do you think about adding a method there and then using it in your spec file?

Reply:
Done